### PR TITLE
Update lists.html

### LINKED ISSUE #6633: The My Lists page should link to instructions to create a list
I'm always frustrated when I go to the [__My Lists__ page](https://openlibrary.org/people/username/lists) but there are no [instructions on how to create a list](https://openlibrary.org/help/faq/reading-log#lists).

### Describe the problem that you'd like solved

https://openlibrary.org/people/username/lists should have link to https://openlibrary.org/help/faq/reading-log#lists
Perhaps the text `No lists yet!` could be a hyperlink to https://openlibrary.org/help/faq/reading-log#lists

### PR DESCRIPTION
**Pull request by Jyotiraditya24**
Closes #6633 

 My Lists page is linking to instructions to create a list

### CODE REVIEW

**Comment by mekarpeles** [comment]
I believe we only want the messaging to the patron who manages the list. Since lists are public, all patrons will see the page, but only the maintainer/owner should see a link to the docs.

File: openlibrary/templates/lists/lists.html [hunk 1]
```diff
@@ -113,8 +113,11 @@
                     id="list-$loop.index">$:render_template("lists/preview", list)</li>
             </ul>
         </div>
+    $elif ctx.user and ctx.user.key != doc.key:
+        <p>$_('This reader hasn't created any lists yet.')</p>
```
Comment (mekarpeles) [inline openlibrary/templates/lists/lists.html:117]:
```suggestion
        <p>$_("This reader hasn't created any lists yet.")</p>
```

File: openlibrary/templates/lists/lists.html [hunk 2]
```diff
@@ -113,8 +113,11 @@
                     id="list-$loop.index">$:render_template("lists/preview", list)</li>
             </ul>
         </div>
+    $elif ctx.user and ctx.user.key != doc.key:
+        <p>$_("This reader hasn't created any lists yet.")</p>
     $else:
-        <p>$_('No lists yet!')</p>
+        <p>$_('You have not created any lists yet.')</p>
```
Comment (mekarpeles) [inline openlibrary/templates/lists/lists.html:119]:
```suggestion
        <p>$_("You haven't created any lists yet.")</p>
```

File: openlibrary/templates/lists/lists.html [hunk 3]
```diff
@@ -113,8 +113,11 @@
                     id="list-$loop.index">$:render_template("lists/preview", list)</li>
             </ul>
         </div>
+    $elif ctx.user and ctx.user.key != doc.key:
+        <p>$_("This reader hasn't created any lists yet.")</p>
     $else:
-        <p>$_('No lists yet!')</p>
+        <p>$_("You haven't created any lists yet.")</p>
+        <p><a href="https://openlibrary.org/help/faq/reading-log#lists">$_('Learn how to create your first list.')</a></p>
```
Comment (mekarpeles) [inline openlibrary/templates/lists/lists.html:120]:
```suggestion
        <p>
          <a href="https://openlibrary.org/help/faq/reading-log#lists"
               target="_blank">$_('Learn how to create your first list.')</a>
        </p>
```

File: openlibrary/templates/lists/lists.html [hunk 4]
```diff
@@ -113,8 +113,14 @@
                     id="list-$loop.index">$:render_template("lists/preview", list)</li>
             </ul>
         </div>
+    $elif ctx.user and ctx.user.key != doc.key:
```
Comment (mekarpeles) [inline openlibrary/templates/lists/lists.html:116]:
```suggestion
    $elif not ctx.user or ctx.user.key != doc.key:
```

**Comment by Jyotiraditya24** [comment]
@mekarpeles thankyou for all your help , this was my first open source contribution

**Pull request merged**